packages/lib/src/dropdown/DropdownMenuItem.tsx

```diff
@@ -3,6 +3,57 @@ import styled from "styled-components";
 import { DropdownMenuItemProps } from "./types";
 import DxcIcon from "../icon/Icon";
 
+const DropdownMenuItemContainer = styled.li<{ visuallyFocused: DropdownMenuItemProps["visuallyFocused"] }>`
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  gap: var(--spacing-gap-xs);
+  min-height: 36px;
+  padding: 0px var(--spacing-padding-xs);
+  cursor: pointer;
+
+  ${(props) =>
+    props.visuallyFocused &&
+    `
+  outline: var(--border-width-m) solid var(--border-color-secondary-medium); 
+  outline-offset: calc(-1 * var(--border-width-m));
+`}
+  &:first-child {
+    border-top-left-radius: var(--border-radius-s);
+    border-top-right-radius: var(--border-radius-s);
+  }
+
+  &:last-child {
+    border-bottom-left-radius: var(--border-radius-s);
+    border-bottom-right-radius: var(--border-radius-s);
+  }
+  &:hover {
```

Comment:
Since both hover and active share the same value, we can just write:
```
  &:hover, &:active {
    background-color: var(--color-bg-neutral-light);
  }
```